### PR TITLE
use expressive variable names to improve readability

### PR DESCRIPTION
**Pull request by atoivonen**

### CODE REVIEW

File: test/create/enableExisting.test.ts [hunk 1]
```diff
@@ -18,38 +18,38 @@ sqs.mockImplementationOnce(() => ({ getQueueUrl }))
 import { enableExisting as ee } from "../../src/create/enableExisting"
```
Comment (LucianoLaratelli) [inline test/create/enableExisting.test.ts:18]: Not a change you made, but--

could this be `enableExisting` without the alias?

File: src/create/mapper.ts [hunk 2]
```diff
@@ -47,18 +47,19 @@ interface IAlarmProps {
   period?: number
   statistic?: string
   threshold?: number
+  treatMissingData?: string
```
[the remark this answers is not in the document]
Comment (atoivonen) [reply src/create/mapper.ts:50]: I didn't intentionally change `stack.ts` 🤔

[the remark this answers is not in the document]
Comment (atoivonen) [reply src/create/mapper.ts]: I rebased with main and these changes were cleaned up

File: test/create/enableExisting.test.ts (hunk 1 above)
Comment (atoivonen) [reply test/create/enableExisting.test.ts]: Good catch! Updated

**Review by lmshepherd — APPROVED** [review]

**Pull request merged**